src/executor/Scheduler.ts

```diff
@@ -233,20 +260,26 @@ export class Scheduler {
             .map(p => p.pluginName);
     }
 
-    private registerWatch(pluginName: string, type: 'ts' | 'less' | 'css'): void {
+    private registerWatch(pluginName: string, type: 'ts' | 'less' | 'css' | 'tsE2E'): void {
         if (!this.watchFiles) {
             return;
         }
 
         const plugin = this.getPlugin(pluginName);
-        const watchDir = path.join(plugin.assetsDir, type);
+        let watchDir = path.join(plugin.assetsDir, type);
```

Comment:
Maybe replace with a `switch(type)` - looks nice even when the code looks same then. See if it makes sense to unify with the switch below for the `jobTracker`